test/boost/row_cache_test.cc

```diff
@@ -4219,3 +4219,58 @@ SEASTAR_TEST_CASE(test_reading_of_nonfull_keys) {
                     .produces(m1);
         });
 }
+
+SEASTAR_TEST_CASE(test_populating_cache_with_expired_and_nonexpired_tombstones) {
+        return do_with_cql_env_thread([](cql_test_env& env) {
```

Comment:
Cosmetics: over-indented, we use 4-space indents.

Reply:
just copypasted indentation from test above but will fix